web/assets/main.scss

```
@@ -26,3 +26,7 @@ $moj-page-width: 1220px;
 .table-row__no-border {
    border-bottom: none;
  }
+
+.colour-text-red {
```

A couple of conventions I'd suggest using here:
- Add `app-` as a prefix to clarify who the style belongs to (compared to `govuk-` and `moj-`)
- Use `!-` as a secondary prefix to highlight that this uses `!important` and will override other rules (this is also [from the Design System](https://design-system.service.gov.uk/styles/layout/#override-how-elements-display))
```suggestion
.app-!-colour-text-red {
```